csc/cmd/controller_unpublish_volume.go

```diff
@@ -27,9 +27,9 @@ USAGE
 	RunE: func(cmd *cobra.Command, args []string) error {
 
 		req := csi.ControllerUnpublishVolumeRequest{
-			Version:         &root.version.Version,
-			NodeId:          controllerUnpublishVolume.nodeID,
-			UserCredentials: root.userCreds,
+			Version: &root.version.Version,
```

Comment:
Hi @sbezverk,

Please ensure that you're using `gofmt` on files prior to committing them. I recommend using [Atom](http://atom.io) with the [Go-Plus plug-in](https://atom.io/packages/go-plus).

Regardless the method, please run `gofmt` on this file in order to correct the formatting.

Reply:
sure thing, since I use linux as workstation in cli mode, I do not have luxury of running atom, but that does not excuse me from forgetting to run gofmt manually, apologies, will do it right away.

Thanks a lot for your review.

Reply:
Hi @sbezverk,

I suggest trying [ElementaryOS](https://elementary.io). I'm using it as a Linux desktop, and I like it very much. It's quite minimal. However, I handle all commits via `vim` and sometimes Go code for a quick change. Because of that I can recommend [vim-go](https://github.com/fatih/vim-go) as a way of making it easy to handle certain common pre-commit activities, such as `gofmt`.